plugins/hw_event/hw_event_plugin.go

```diff
@@ -189,9 +101,11 @@ func publishHwEvent(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 	bodyBytes, err := ioutil.ReadAll(r.Body)
 	if err != nil {
-		log.Errorf("error reading hw event %v", err)
+		log.Errorf("error reading hw event: %v", err)
 		return
 	}
+	// Use time received to calculate lapse
+	timeReceived := types.Timestamp{Time: time.Now().UTC()}.Time
```

Comment:
this is not time received, it should be time event created

Reply:
updated